{{ cookiecutter.project_name }}/requirements.txt

```diff
@@ -55,14 +51,6 @@ django-csp==2.0.3
 # sha256: k_gRIthpg8uwVs0Syn3jnXYLbz4wYOQ-_1xeIDRRb6I
 pytz==2015.2
 
-# sha256: eLAxFrrdDbsu1N7clqSj1CE4uUu4n8Lrmffzvf0Zn1Y
-# sha256: zIrr3sWl-5iZEvFX93s8IaXi8tpiOvkKe0drEGqDSr8
-# sha256: drxjpOLV5aDfd8p9GPD1bixGz7YrcRA7qSqSx5-rHgM
-nose==1.3.4
```

Comment:
This causes a [test failure](https://travis-ci.org/mozilla/sugardough/jobs/57331269) but I'm not opposed to removing it if you want to replace the `eq_` shorthand we get from `nose.tools`.